src/templates/test/mocks/MockFunctionParametersScenario.mustache

```diff
@@ -0,0 +1,82 @@
+pragma solidity 0.4.24;
+
+
+contract {{ name.Plural }}MockFunctionParametersScenario {
+    {{#each variables}}
+    {{ type }}[] private unknownLenght{{ name.UpperCamelCase }};
+    {{/each}}
+
+    function singleStruct(
+        {{#each variables}}
+        {{ type }} _{{ name.lowerCamelCase }}{{#unless @last}},{{/unless}}
+        {{/each}}
+    ) public pure returns(
+        {{#each variables}}
+        {{ type }}{{#unless @last}},{{/unless}}
+        {{/each}}
+    ) {
+        {{#each variables}}
+        {{ type }} {{ name.lowerCamelCase }} = _{{ name.lowerCamelCase }};
+        {{/each}}
+        return (
+            {{#each variables}}
+            {{ name.lowerCamelCase }}{{#unless @last}},{{/unless}}
+            {{/each}}
+        );
+    }
+
+    function knownLengthStructArray(
+        {{#each variables}}
+        {{ type }}[10] _{{ name.lowerCamelCase }}{{#unless @last}},{{/unless}}
```

Comment:
Not sure if it's worth doing, but maybe this 10 should be set to a constant variable so it could be changed easier in the future, it's used in a couple of places